app/services/sentiment_analysis_service.rb

```diff
@@ -0,0 +1,7 @@
+class SentimentAnalysisService
+  def determine_sentiment(*phrases)
```

Comment:
Author:  Its called a [splat](http://blog.honeybadger.io/ruby-splat-array-manipulation-destructuring/) & its used for destructuring & manipulating arrays